CMakeLists.txt

```diff
@@ -32,12 +32,12 @@ SET(PROJECT_DESCRIPTION "${PROJECT_NAME_UCASE} - Free music production software"
 SET(PROJECT_COPYRIGHT   "2008-${PROJECT_YEAR} ${PROJECT_AUTHOR}")
 SET(VERSION_MAJOR       "1")
 SET(VERSION_MINOR       "1")
-SET(VERSION_PATCH       "91")
-#SET(VERSION_SUFFIX     "")
-SET(VERSION             "${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH}")
-IF(VERSION_SUFFIX)
-	SET(VERSION     "${VERSION}-${VERSION_SUFFIX}")
-ENDIF(VERSION_SUFFIX)
+SET(VERSION_BUILD       "91")
```

Comment:
P.S. Off-topic, but @jasp00 FYI, squashing your commits each time you make a change can make it hard to find these comments as it breaks the links that go out in email.

> When version 1.2.0 is released, will you immediately tag 1.3.0-alpha? The alpha tag could be configured not to be deployed.

Historically we leave the master with the wrong version number.  I don't think this is ideal at all as packagers get very confused as to which branch contains which version.  I'd like to know @Lukas-W's thoughts on the topic, but I don't think we need to specifically `tag` anything but rather change the internal versioning to avoid confusion down the road.

> If version is 1.2.0-rc2, what should be done about FILEVERSION numbers? Should they be dropped since a FileVersion string is also provided?

I'm not sure exactly the question, but I think it's in regards to the win32 `File version` string, which is of marginal importance on Windows.

Currently the most important versioning information on Windows is stored in Add/Remove Programs and within the Application's own About dialog, which I would consider the bare minimum.

NSIS isn't given this information explicitly which leads me to believe it's all set using [`CPACK` variables](https://github.com/LMMS/lmms/blob/master/cmake/CMakeLists.txt#L5:L7) and then [cpack passes it off to NSIS]((https://cmake.org/Wiki/CMake:CPackConfiguration).  This also explains where the original `MAJOR`, `MINOR`, `PATCH` naming came from.